src/databricks/labs/ucx/source_code/graph.py

```diff
@@ -185,11 +185,16 @@ def _process_node(self, base_node: NodeBase):
         if isinstance(base_node, SysPathChange):
             self._mutate_path_lookup(base_node)
         if isinstance(base_node, NotebookRunCall):
-            strpath = base_node.get_notebook_path()
-            if strpath is None:
-                yield DependencyProblem('dependency-not-constant', "Can't check dependency not provided as a constant")
+            paths = base_node.get_notebook_paths()
+            if None in paths:
```

Comment:
We don't want to try to register the not `None` paths?

Reply:
good point